packages/components/tests/dummy/app/templates/components/reveal.hbs

```diff
@@ -0,0 +1,153 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{page-title "Reveal Component"}}
+
+<Shw::Text::H1>Reveal</Shw::Text::H1>
+
+<section data-test-percy>
+  <Shw::Text::H2>Content</Shw::Text::H2>
+
+  <Shw::Text::H3>On its own</Shw::Text::H3>
+
+  <Shw::Flex {{style gap="1.5rem"}} @direction="column" as |SF|>
+    <SF.Item @label="With plain text content">
+      <Hds::Reveal @text="Developer instructions">
+        Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to
+        corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the
+        holistic world view of disruptive innovation via workplace diversity and empowerment.
+      </Hds::Reveal>
+    </SF.Item>
+
+    <SF.Item @label="With HTML formatted content (containing alert & list)">
+      <Hds::Reveal @text="Show me how">
+        <Hds::Alert @type="compact" as |A|>
+          <A.Description>Instructions to add the verification record to your domain host</A.Description>
+        </Hds::Alert>
+        <ol>
+          <li>
+            For the record type, select
+            <b>TXT</b>.
+          </li>
+          <li>
+            In the
+            <b>Name/Host/Alias</b>
+            field, enter
+            <b>@</b>
+            or leave blank. Your host might require you to enter your domain, which looks like
+            <b>example.com</b>
+          </li>
+          <li>
+            In the
+            <b>Time to Live (TTL)</b>
+            field, enter
+            <b>86400</b>
+            or leave the default.
+          </li>
+          <li>
+            In the
+            <b>Value/Answer/Destination</b>
+            field, paste the verification record you copied.
+          </li>
+          <li>
+            Save the record.
+          </li>
+        </ol>
+      </Hds::Reveal>
+    </SF.Item>
+
+    <SF.Item @label="With interactive content">
+      <Hds::Reveal @text="More options">
+        See here for more info:
+        <a href="https://www.hashicorp.com/" target="_blank" rel="noopener noreferrer">HashiCorp</a>
+      </Hds::Reveal>
+    </SF.Item>
+  </Shw::Flex>
+
+  <Shw::Text::H3>Used next to other content</Shw::Text::H3>
+
+  <Shw::Flex @direction="column" {{style gap="1rem"}} as |SF|>
+    <SF.Item @label="Used beneath a form input">
+      <Hds::Form::TextInput::Field as |F|>
+        <F.Label>Private subnet 1</F.Label>
+        <F.HelperText>Ensure that the private subnets have access to the external network.
+        </F.HelperText>
+      </Hds::Form::TextInput::Field>
+      <Hds::Reveal @text="Where can I find this?">
+        <Hds::Alert @type="compact" as |A|>
+          <A.Description>Navigate to Subnets(1), if you already have subnet created for the VPC of choice, copy the
+            Subnet ID(2). Otherwise, Create subnet(3) for chosen VPC.</A.Description>
+        </Hds::Alert>
+        <p>
+          <img
```

Comment:
this image should have a special "local" class (specific for this page) that sets the max width to 100% (same as it would happen on the consumers side)